test/performance-test-suite/pkg/benchmarks/writetxs/benchmark.go

```diff
@@ -130,6 +132,36 @@ func (b *benchmark) Warmup() error {
 		return err
 	}
 
+	if b.cfg.Replica == "async" || b.cfg.Replica == "sync" {
+		replicaDirName := fmt.Sprintf("%s-replica-tx-test", b.cfg.Replica)
+
+		replicaOptions := server.
+			ReplicationOptions{}
+
+		options2 := server.
+			DefaultOptions().
+			WithDir(replicaDirName).
+			WithLogFormat(logger.LogFormatJSON)
+
+		if b.cfg.Replica == "async" {
+			options2.
+				WithReplicationOptions(replicaOptions.WithIsReplica(true))
+		}
+
+		if b.cfg.Replica == "sync" {
+			options2.
+				WithReplicationOptions(replicaOptions.WithIsReplica(true).WithSyncReplication(true))
```

Comment:
primary server also needs to be configured for sync replication

e.g. `WithReplicationOptions(replicaOptions.WithIsReplica(false).WithSyncReplication(true).WithSyncAcks(1))`